.github/workflows/rust.yml

```diff
@@ -38,20 +41,35 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v4
+      - uses: dtolnay/rust-toolchain@stable
+        with:
+          components: clippy
+      - uses: taiki-e/install-action@v2
+        with:
+          tool: cargo-nextest
       - uses: Swatinem/rust-cache@v2
-      - run: cd ${{ matrix.package }} && cargo test --all-targets
+      - run: cd ${{ matrix.package }} && cargo nextest run
       - run: cd ${{ matrix.package }} && cargo clippy --all-targets -- -D warnings
 
-  Lint:
+  Rustfmt:
     runs-on: ubuntu-latest
     steps:
+      # No need to test `rinja_derive_standalone`. It has same the `src` folder as `rinja_derive`.
+      # No need to run the checks in parallel. They are fast enough.
+      # No need for caching. No code is compiled at all.
       - uses: actions/checkout@v4
-      - uses: dtolnay/rust-toolchain@stable
+      - uses: dtolnay/rust-toolchain@nightly
         with:
-          components: rustfmt, clippy
-      - uses: Swatinem/rust-cache@v2
-      - run: cargo fmt --all -- --check
-      - run: cargo clippy --all-targets -- -D warnings
+          components: rustfmt
+      - run: cd rinja                  && cargo fmt -- --check
```

Comment:
Instead of doing it like this, why not using matrix so then we list all folders we want to go into so we can just have `run: cd $FOLDER && cargo fmt -- --check`?

Reply:
> No need to run the checks in parallel. They are fast enough.

Even though the runners are free for open source projects, I don't think we should use more CPU time than we need to. Be eco friendly and such.

Also, we already have 17 checkmarks on each PR. At some point it gets annoying to scroll until you find the red cross on the failed test. :)

Reply:
Can't have that not in parallel?

Reply:
I don't know if there is a "run is one `run:` instruction multiple times with different env vars" syntax?

Reply:
`run` is shell command, so we can likely have an array and iterate over it? (But I suppose it'd need a `cd -` command in addition, I suppose it's fine)

Reply:
Yeah, should have thought about that! :) A formatting error will be reported like that: <https://github.com/rinja-rs/rinja/actions/runs/9905823875/job/27366138342?pr=69>

Reply:
Sounds perfectly fine. :+1: